.gitignore

```diff
@@ -215,3 +215,6 @@ cython_debug/
 !crates/ruff_python_resolver/resources/test/airflow/venv/lib
 !crates/ruff_python_resolver/resources/test/airflow/venv/lib/python3.11/site-packages/_watchdog_fsevents.cpython-311-darwin.so
 !crates/ruff_python_resolver/resources/test/airflow/venv/lib/python3.11/site-packages/orjson/orjson.cpython-311-darwin.so
+
+# Ignore the zipped version of typeshed that we create at build time
+crates/red_knot/vendor/zipped_typeshed.zip
```

Comment:
Can we create the `zip` in the `target` directory where cargo stores all its build artifacts? This way, we can avoid adding an additional ignore, and it also makes it clear that this is a build-time-created artifact.

Reply:
To clarify: do you mean `$ROOT/target` (which already exists), or do you mean creating a new `$ROOT/crates/red_knot/target` directory for redknot-specific build artifacts?

Reply:
Reading through https://github.com/rust-lang/cargo/issues/9661, it sounds like getting the location of the `target` directory reliably from a build script is a deliberately unsolved problem, and that build scripts are not meant to put artifacts there. https://github.com/rust-lang/cargo/issues/9661 says:

> `OUT_DIR` — If the package has a build script, this is set to the folder where the build script should place its output. See below for more information. (Only set during compilation.)

That implies to me that I should be putting the zip there, rather than where I have it or in `target/`

Reply:
Yeah sorry, I'm not very familiar with build scripts. Out dir seems to be what I was looking for.